member_type.go

```diff
@@ -0,0 +1,12 @@
+package zoom
+
+// Member represents an group member
+type Member struct {
+	ID        string `json:"id"`
+	Email     string `json:"email"`
+}
+
+type ResopnseAddGroupMembers struct {
+	Ids		 string `json:"ids"`
```

Comment:
```suggestion
	IDs		 string `json:"ids"`
```

Reply:
The linter is going to complain, this struct needs a comment. The Zoom docs don't say what format IDs takes - is it comma-delimited? Could you please do a test where you make this API call against a real account to add multiple members and include in your comment an example of what this data looks like?

Reply:
API return string has comma-delimited.
I'll add these comment!